.github/actions/api/breaking-change/action.yaml

```
@@ -0,0 +1,24 @@
+name: 'OpenAPI Spec: check for breaking API changes'
+description: 'Detect breaking changes in OpenAPI Specification 3'
+inputs:
+  base:
+    description: 'Path of original OpenAPI spec in YAML or JSON format'
+    required: true
+  revision:
+    description: 'Path of revised OpenAPI spec in YAML or JSON format'
```

I understand _why_ they're called "base" and "revision", but the latter is also [what Git calls a version](https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection).  Can we find another name for the arg?

Perhaps

```suggestion
  revised:
    description: 'Path of revised OpenAPI spec in YAML or JSON format'
```

or similarly "changed", "new", "upgraded"?